src/lib/helper/arch_compat.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#ifdef __SSE__
+#  include <immintrin.h>
+#  define PAUSE() _mm_pause()
+#else
+#  define PAUSE()
+#endif
+
+#ifdef __powerpc64__
+#  define __builtin_ia32_sfence()
```

Comment:
sfence is actually used only in the nvm-version but nowhere in hyrise. However I've made it consistent with the `PAUSE`-macro above.